bal_addresses/pools_gauges.py

```diff
@@ -77,6 +77,30 @@ def query_preferential_gauges(self, skip=0, step_size=100) -> list:
             result += self.query_preferential_gauges(skip + step_size, step_size)
         return result
 
+    def query_root_gauges(self, skip=0, step_size=100) -> list:
```

Comment:
We should refactor this at some point to use the whole query system we just implemented.  If you'd prefer to get this merged can make a ticket to do it later to resolve this.